### PR TITLE
Fix #1341 - Make sure initializationOptions is at least empty object

### LINKED ISSUE #1341: initializationOptions: Cannot read property 'config' of undefined
Hi, i'm trying to use vue-language-server with vim and LanguageClient-neovim and it does not work.

I traced bug to this error:
```
00:54:01 WARN unnamed src/language_server_protocol.rs:2132 Failed to start language server automatically. Error: Failure { jsonrpc: Some(V2), error: Error { code: InternalError, message: "Request initialize failed with message: Cannot read property \'config\' of undefined", data: None }, id: Num(1) }
```
https://github.com/autozimu/LanguageClient-neovim/blob/next/src/language_server_protocol.rs#L897

Looks like initializationOptions can be null, but vue-language-server does not take this into account here:

https://github.com/vuejs/vetur/blob/82d16031f58e0f54fde1fce94cf2c9518fe5080e/server/src/services/vls.ts#L83

It looks like trivial issue, can you fix it or should i make pull-request?

### PR DESCRIPTION
**Pull request by zulus**
Simple change that allows me to register vls in eclipse LSP4E

### CODE REVIEW

File: server/src/config.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export interface VLSFullConfig extends VLSConfig {
   stylusSupremacy?: any;
 }
 
-export function getDefaultVLSConfig(): VLSConfig {
+export function getDefaultVLSConfig(): VLSFullConfig {
```
Comment (zulus) [inline server/src/config.ts:77]: Eclipse LSP4E wasn't able start because normally this field just not exists.

File: server/src/services/vls.ts [hunk 2]
```diff
@@ -83,6 +83,7 @@ export class VLS {
   }
 
   async init(params: InitializeParams) {
+    params.initializationOptions = params.initializationOptions || {};
```
Comment (zulus) [inline server/src/services/vls.ts:86]: Required to use VLS in Eclipse LSP4E, probably other simple implementation also. This field normally not exists.

File: server/src/config.ts [hunk 3]
```diff
@@ -118,6 +118,18 @@ export function getDefaultVLSConfig(): VLSConfig {
       experimental: {
         templateInterpolationService: false
       }
-    }
+    },
+    css: {},
```
Comment (zulus) [inline server/src/config.ts:122]: VLS just forward config.emmet and others to sub-services. Even undefined, after this sass `<style lang="scss">` is operational.

**Comment by octref** [comment]
No longer applies, and should already be fixed by #1705.

File: server/src/config.ts (hunk 3 above)
Comment (octref) [reply server/src/config.ts:122]: OK, will take this PR then

File: server/src/services/vls.ts (hunk 2 above)
Comment (octref) [reply server/src/services/vls.ts:86]: No longer needed since we now use `?.` to access it

**Comment by octref** [comment]
Thank you and sorry for the delay 👍

**Pull request merged**